### PR TITLE
Added empty service

### PR DESCRIPTION
**Pull request by maybee16**
https://trello.com/c/Jd8JE294/117-reactionservice

### CODE REVIEW

File: Common.Configurations.targets [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(AssemblyName.Contains('.UnitTests'))">
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
+	<PackageReference Include="DigitalOffice.Kernel" Version="1.0.5" />
+	<PackageReference Include="DigitalOffice.Kernel.BrokerSupport" Version="1.0.0" />
+	<PackageReference Include="DigitalOffice.Kernel.ImageSupport" Version="1.0.2" />
```
Comment (vorobevamv) [inline Common.Configurations.targets]: 1.0.3

File: Common.Configurations.targets [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	<PackageReference Include="DigitalOffice.Kernel.EFSupport" Version="1.0.0" />
+	<PackageReference Include="DigitalOffice.Models" Version="1.0.6" />
```
Comment (vorobevamv) [inline Common.Configurations.targets]: 1.0.7

File: src/ReactionService/appsettings.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      "WithThreadId"
+    ],
+    "WriteTo": [
+      {
+        "Name": "Console"
+      },
+      {
+        "Name": "Seq",
+        "Args": {
+          "serverUrl": "http://localhost:9864",
```
Comment (vorobevamv) [inline src/ReactionService/appsettings.json:19]: http://localhost:5341

Comment (vorobevamv) [inline src/ReactionService/appsettings.json]: "serverUrl": "http://localhost:5341"

it's not ReactionService localhost, it's Seq

File: src/ReactionService/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    ]
+  },
+  "ServiceInfo": {
+    "Id": "D1522472-9BE6-4AFD-98B2-A7F4FA2A2D2E",
+    "Name": "ReactionService"
+  },
+  "RabbitMQ": {
+    // UserService
+    "CheckUserIsAdminEndpoint": "UserService_GetUserAdminStatus",
+    "GetUsersDataEndpoint": "UserService_GetUsersData",
```
Comment (vorobevamv) [inline src/ReactionService/appsettings.json]: what for?

File: src/ReactionService/appsettings.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+    // AuthService
+    "ValidateTokenEndpoint": "AuthService_ValidateToken",
+
+    //RightService
+    "CheckUserRightsEndpoint": "CRService_CheckUserRights",
+
+    //ImageService
+    "GetImagesEndpoint": "ImageService_GetImages"
```
Comment (vorobevamv) [inline src/ReactionService/appsettings.json]: add "CreateImagesEndpoint": "ImageService_CreateImages"

File: src/ReactionService.Models.Dto/Configurations/RabbitMqConfig.cs [hunk 6]
```diff
@@ -0,0 +1,11 @@
+﻿using LT.DigitalOffice.Kernel.BrokerSupport.Attributes;
+using LT.DigitalOffice.Kernel.BrokerSupport.Configurations;
+using LT.DigitalOffice.Models.Broker.Requests.User;
+
+namespace LT.DigitalOffice.ReactionService.Models.Dto.Configurations;
+
+public class RabbitMqConfig : BaseRabbitMqConfig
+{
+  [AutoInjectRequest(typeof(IGetUsersDataRequest))]
+  public string GetUsersDataEndpoint { get; set; }
```
Comment (vorobevamv) [inline src/ReactionService.Models.Dto/Configurations/RabbitMqConfig.cs]: do you need it?

File: Common.Configurations.targets (hunk 1 above)
Comment (maybee16) [reply Common.Configurations.targets]: Done

File: Common.Configurations.targets (hunk 2 above)
Comment (maybee16) [reply Common.Configurations.targets]: Done

File: src/ReactionService/appsettings.json (hunk 3 above)
Comment (maybee16) [reply src/ReactionService/appsettings.json]: Done

File: src/ReactionService/appsettings.json (hunk 4 above)
Comment (maybee16) [reply src/ReactionService/appsettings.json]: Done

File: src/ReactionService/appsettings.json (hunk 5 above)
Comment (maybee16) [reply src/ReactionService/appsettings.json]: Done

File: src/ReactionService.Models.Dto/Configurations/RabbitMqConfig.cs [hunk 7]
```diff
@@ -1,11 +1,16 @@
...
+using LT.DigitalOffice.Models.Broker.Requests.Image;
 
 namespace LT.DigitalOffice.ReactionService.Models.Dto.Configurations;
 
 public class RabbitMqConfig : BaseRabbitMqConfig
 {
-  [AutoInjectRequest(typeof(IGetUsersDataRequest))]
-  public string GetUsersDataEndpoint { get; set; }
+  // Image
+
```
Comment (vorobevamv) [inline src/ReactionService.Models.Dto/Configurations/RabbitMqConfig.cs:10]: remove empty line

File: src/ReactionService.Models.Dto/Configurations/RabbitMqConfig.cs [hunk 8]
```diff
@@ -1,11 +1,16 @@
...
+  [AutoInjectRequest(typeof(IGetImagesRequest))]
+  public string GetImagesEndpoint { get; set; }
+
+  [AutoInjectRequest(typeof(ICreateImagesRequest))]
+  public string CreateImagesEndpoint { get; set; }
```
Comment (vorobevamv) [inline src/ReactionService.Models.Dto/Configurations/RabbitMqConfig.cs:15]: usually add endpoints when start using them. it's not mistake, let it be, but then make sure you use them

**Review by vorobevamv — APPROVED** [review]

File: src/ReactionService.Models.Dto/Configurations/RabbitMqConfig.cs (hunk 7 above)
Comment (maybee16) [reply src/ReactionService.Models.Dto/Configurations/RabbitMqConfig.cs:10]: Done

File: src/ReactionService.Data/ReactionService.Data.csproj [hunk 9]
```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
+
+  <ItemGroup>
+	<ProjectReference Include="..\ReactionService.Data.Provider.MsSql.Ef\ReactionService.Data.Provider.MsSql.Ef.csproj" />
```
Comment (TIove) [inline src/ReactionService.Data/ReactionService.Data.csproj]: This spaces happens, 'cause you use tabs on lines 6 and 7, but spaces on 8. In VS/Rider this looks the same, but better to change this all to tabs or spaces (depends on your religion).

Comment (TIove) [reply src/ReactionService.Data/ReactionService.Data.csproj]: I prefer spaces here, 'cause everywhere else we add sapces in csproj files

File: src/ReactionService.Models.Dto/ReactionService.Models.Dto.csproj [hunk 10]
```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
+
+  <ItemGroup>
+	<PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
+  </ItemGroup>
+
+  <ItemGroup>
+	<ProjectReference Include="..\ReactionService.Models.Db\ReactionService.Models.Db.csproj" />
```
Comment (TIove) [inline src/ReactionService.Models.Dto/ReactionService.Models.Dto.csproj]: Same as before, use spaces

File: src/ReactionService.Models.Dto/ReactionService.Models.Dto.csproj [hunk 11]
```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
+
+  <ItemGroup>
+	<PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
```
Comment (TIove) [inline src/ReactionService.Models.Dto/ReactionService.Models.Dto.csproj]: Same as before, use spaces

File: src/ReactionService/ReactionService.csproj [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+  <ItemGroup>
+	<PackageReference Include="AspNetCore.HealthChecks.SqlServer" Version="6.0.2" />
+	<PackageReference Include="AspNetCore.HealthChecks.UI.Client" Version="6.0.5" />
+	<PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.8" />
+	<PackageReference Include="Serilog.AspNetCore" Version="6.0.1" />
+	<PackageReference Include="Serilog.Enrichers.Environment" Version="2.2.0" />
+	<PackageReference Include="Serilog.Enrichers.Process" Version="2.0.2" />
+	<PackageReference Include="Serilog.Enrichers.Thread" Version="3.1.0" />
+	<PackageReference Include="Serilog.Settings.Configuration" Version="3.3.0" />
+	<PackageReference Include="Serilog.Sinks.Seq" Version="5.1.1" />
```
Comment (TIove) [inline src/ReactionService/ReactionService.csproj]: Spaces or tabs everywhere

File: .editorconfig [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+#prefer fields not to be prefaced with this. or Me. in Visual Basic
+dotnet_style_qualification_for_field = false:suggestion
+#prefer methods not to be prefaced with this. or Me. in Visual Basic
+dotnet_style_qualification_for_method = false:suggestion
+#prefer properties not to be prefaced with this. or Me. in Visual Basic
+dotnet_style_qualification_for_property = false:suggestion
+
+[*.{cs,vb}]
+tab_width=2
+indent_size=2
```
Comment (TIove) [inline .editorconfig]: I would add the next strings, but depends on @vorobevamv 
```suggestion
indent_size=2

[*.{csproj,sln}]
indent_style = space
indent_size=2
tab_width=2
```

**Review by Vasis3038 — APPROVED** [review]

File: src/ReactionService.Data/ReactionService.Data.csproj (hunk 9 above)
Comment (maybee16) [reply src/ReactionService.Data/ReactionService.Data.csproj]: Done

File: src/ReactionService.Models.Dto/ReactionService.Models.Dto.csproj (hunk 10 above)
Comment (maybee16) [reply src/ReactionService.Models.Dto/ReactionService.Models.Dto.csproj]: Done

File: src/ReactionService.Models.Dto/ReactionService.Models.Dto.csproj (hunk 11 above)
Comment (maybee16) [reply src/ReactionService.Models.Dto/ReactionService.Models.Dto.csproj]: Done

File: src/ReactionService/ReactionService.csproj (hunk 12 above)
Comment (maybee16) [reply src/ReactionService/ReactionService.csproj]: Done

**Review by TIove — APPROVED** [review]

**Review by AlfaRedDog — APPROVED** [review]

**Pull request merged**